### DISCUSSION TITLE
Open Accordion/tabs with an Anchor link

### DISCUSSION THREAD

**Discussion by portrayaloflife** [elementor/elementor#24770] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Have had a lot of clients ask for this functionality for them to be able to use an anchor link or for a button to open and scroll to an accordion or tab element. 

### Describe the solution you'd like.

The ability to add an ID to an anchor or tab element that opens when linked to via an anchor

### Describe alternatives you've considered.

Scripts, other 3rd party plugins

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by mkarim1378** [comment]
I really need this feature. Thanks for mention!
And if anyone knows a way to do that please comment here.
